src/api.ts

```diff
@@ -0,0 +1,15 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getData = async(url: string) => {
```

Comment:
Would be great to have return type here

Reply:
Agree but how to check it with our linter